PIM/API/01_Overview.md

```diff
@@ -0,0 +1,227 @@
+
+
+#### Headers
+
+Headers contain metadata in key-value pairs that are sent along with HTTP requests and responses. They are essential for the API calls, as they provide additional information for communication between client and server.
+
+The following headers are required for communication with the *Actindo Core1 OpenAPI*:
+
+| Key | Value |
+|-----|-------|
+| Content-Type  | application/json |
+| Accept        | application/json |
+| Authorization | access token <br> See [Generate an access token](./02_GetStarted.md#step-3-generate-an-access-token). |
+
+[comment]: <> (Unsicher, ob das stimmt, und wie authentication/authorization via header/request body funktioniert)
+
+
+#### Methods
+
+The *POST* method is always used for all *Actindo Core1 OpenAPI* endpoints. All other methods are not relevant for communication with the *Actindo Core1 Platform*.
+
+
+#### Format
+
+The *Actindo Core1 OpenAPI* only accepts and returns JSON body requests. 
+
+
+### Response bodies
+
+An API response is a reply to an API call. In other words, for every request body sent, a response body is returned. 
+
+In the *Actindo Core1 OpenAPI*, the response body contains a JSON object, which always includes the boolean field `"success"` indicating if the request has been successful (`true`) or has failed (`false`). 
+
+In a successful call, the response includes the information requested or the details of the resources created, edited, or deleted. In an unsuccessful call, the response includes an error code and at least a brief description of the reason why the call failed. 
+
+The following examples display two response samples. These response samples are not exhaustive, that is, some lines of code have been omitted for the sake of brevity and replaced with an ellipsis (...). Bear also in mind that the response sample varies depending on the endpoint and the resource addressed.  
+
+#### Sample: Success, product created
+
+        {
+            "product": {
+            "id": 1171,
+            "sku": "new-product",
+            "attributeSetId": 102,
+            ...
+            },
+            "success": true,
+            "displayMessage": "The product was successfully created",
+            "displayMessageTitle": "Create product",
+            "error": null,
+            "job_id": null
+        }
+
+> [Info] For a complete list of attributes contained in the product object, see [The product object](./05_ManageProducts.md#the-product-object).
+
+#### Sample: Error, required field missing or empty
+
+        {
+            "success": false,
+            "errno": 10023,
+            "error": "Product could not be created: Attribute set id is not set or empty",
+            "trace": "<pre> (Actindo\\Modules\\Actindo\\PIM\\Exception\\CreateProductFailedException) in ... \/pre>"
+        }
+
+> [Info] For a response schema, refer to the API documentation in your *Actindo Core1 Platform* instance under *Dev Tools > API > Module name > API endpoint*.
```

Comment:
Unter dem Pfad finde ich auf Anhieb erst mal  nichts... Was meinst Du genau? Einen API Endpunkt auswählen und nach unten scrollen? und dann in der Mitte? Vielleicht einen Screenshot?
Dev Tools > API > "Module name" > API endpoint